src/shared/constants.ts

```diff
@@ -38,6 +38,10 @@ export enum WalletDefaultNetworkConfigurationIds {
 
 export type DefaultNetworkConfigurations = keyof typeof WalletDefaultNetworkConfigurationIds;
 
+const supportedBlockchains = ['stacks', 'bitcoin'] as const;
```

Comment:
`enum` ?

Reply:
Enums can look cleaner, but require another import just to use a value. 

I've moved away from using them in some cases, as with correct string literal typing, you're prohibited from making a mistake/get better ts autocompletion than we enums.

Happy to have a conversation though as to whether we can set rules on when we should/shouldn't use them.

Reply:
We do already have a `Blockchains` type in our models. We should use or kill that if adding a new `SupportedBlockchains` type.

Reply:
I just like `enum`s for the type suggestions in the IDE. I think using the existing `Blockchains` type makes sense here though.

I prefer the import over the typo

Reply:
<img width="766" alt="image" src="https://github.com/hirosystems/wallet/assets/1618764/6cc34ae2-ca80-45af-bc77-0bc55d46f36e">

You can strongly type to a string literal though, so typos not possible.

Reply:
I lost him at Angular 🙄  😂